knowledge-base/how-to-use-reportsource-objects-with-reportbook.md

```diff
@@ -0,0 +1,55 @@
+---
+title: How to use ReportSource objects with ReportBook
+description: How to use ReportSource objects with ReportBook.
+type: how-to
+page_title: How to use ReportSource objects with ReportBook
+slug: how-to-use-reportsource-objects-with-reportbook
+res_type: kb
+---
+
+## Description
+ 
+Adding reports to [ReportBook](../designing-reports-general-explanation) shows a warning in **Visual Studio**.  
+*"The Reports property is now obsolete. Please, use ReportSources property instead."*
+ 
+## Solution  
+  
+As of the **R1 2017** release, adding reports to the [ReportBook.Reports](../p-telerik-reporting-reportbook-reports) collection is **obsolete** - [API Breaking Changes](../upgrade-path-2017-r1#api-breaking-changes).  
+  
+To allow integration with Standalone Designer, ReportBook was updated to use [ReportSource](../report-sources) objects for adding the reports.  
+  
+An updated approach includes adding the necessary **ReportSources** to the [**ReportBook.ReportSources**](../p-telerik-reporting-reportbook-reportsources) collection.     
+  
```

Comment:
Add some lines explaining that there are 2 solutions, one for versions prior to R1 2017 and one for later versions. Also mark that in the environment table as well, as the warning is visible only for versions after making the functionality obsolete.